plugins/available/base.plugin.bash

```diff
@@ -167,10 +157,11 @@ function command_exists ()
     param '1: command to check'
     example '$ command_exists ls && echo exists'
```

Comment:
why is there even another `command_exists` here?
lets just remove it instead

Reply:
Removed! 👍